src/main/java/com/google/devtools/build/lib/bazel/bzlmod/InterimModule.java

```diff
@@ -160,6 +160,14 @@ public abstract static class Builder {
 
     abstract ImmutableList.Builder<String> bazelCompatibilityBuilder();
 
+    abstract ImmutableList.Builder<String> flagAliasesBuilder();
+
+    @CanIgnoreReturnValue
+    public final Builder addFlagAlias(String nativeName, String starlarkLabel) {
+      flagAliasesBuilder().add(String.format("%s=%s", nativeName, starlarkLabel));
```

Comment:
The label string needs to be converted to a Label and then turned into a string via `getUnambiguousCanonicalForm`, otherwise it will be reinterpreted relative to the main repo

Reply:
That's a different one, I think: you have to set a default value in the `builder()` method.